pinot-common/src/main/java/org/apache/pinot/common/metrics/ServerGauge.java

```diff
@@ -48,7 +48,8 @@ public enum ServerGauge implements AbstractMetrics.Gauge {
   JVM_HEAP_USED_BYTES("bytes", true),
   // Ingestion delay metrics
   REALTIME_INGESTION_DELAY_MS("milliseconds", false),
-  END_TO_END_REALTIME_INGESTION_DELAY_MS("milliseconds", false);
+  END_TO_END_REALTIME_INGESTION_DELAY_MS("milliseconds", false),
+  UPSERT_VALID_DOC_ID_SNAPSHOT_COUNT("upsertValidDocIdSnapshotCount", false);
```

Comment:
can you add a description along with this metric enum?
See example [here](https://github.com/apache/pinot/blob/master/pinot-common/src/main/java/org/apache/pinot/common/metrics/ServerTimer.java#L29)

Reply:
@KKcorps is there a reason why you are not adding it as a enum field and simply a java comment?